config/webpack.target.mobile.js

```diff
@@ -15,6 +15,17 @@ module.exports = function(production, app) {
       path: path.resolve(__dirname, `../targets/${app}/mobile/www`),
       filename: '[name].js'
     },
+    module: {
+      rules: [
+        {
+            test: /\.(eot|svg|ttf|woff|woff2)$/,
```

Comment:
Why did you add `svg` file?

Reply:
Same question for `ttf`, `eot` ;)

Reply:
I felt like it would be a good idea to cover all possible font formats, but I think I should at least remove svg now that you say it. We use it for loads of different things, better not mix up the loaders.

Reply:
> Don’t write code in anticipation of features that haven’t been requested. You aren’t going to need it (YAGNI).